keyboards/novelkeys/nk_classic_tkl_iso/keyboard.json

```
@@ -0,0 +1,272 @@
+            {"x": 218, "y": 34, "flags": 9},
+            {"x": 211, "y": 34, "flags": 9},
+            {"x": 205, "y": 34, "flags": 9},
+            {"x": 198, "y": 34, "flags": 9},
+            {"x": 192, "y": 34, "flags": 9},
+            {"matrix": [3, 12], "x": 163, "y": 40, "flags": 4},
+            {"matrix": [3, 11], "x": 150, "y": 40, "flags": 4},
+            {"matrix": [3, 10], "x": 138, "y": 40, "flags": 4},
+            {"matrix": [3, 9], "x": 125, "y": 40, "flags": 4},
+            {"matrix": [3, 8], "x": 112, "y": 40, "flags": 4},
+            {"matrix": [3, 7], "x": 99, "y": 40, "flags": 4},
+            {"matrix": [3, 6], "x": 86, "y": 40, "flags": 4},
+            {"matrix": [3, 5], "x": 74, "y": 40, "flags": 4},
+            {"matrix": [3, 4], "x": 61, "y": 40, "flags": 4},
+            {"matrix": [3, 3], "x": 48, "y": 40, "flags": 4},
+            {"matrix": [3, 2], "x": 35, "y": 40, "flags": 4},
+            {"matrix": [3, 1], "x": 22, "y": 40, "flags": 4},
+            {"matrix": [3, 0], "x": 5, "y": 40, "flags": 1},
+            {"matrix": [4, 0], "x": 1, "y": 52, "flags": 1},
+            {"matrix": [4, 1], "x": 16, "y": 52, "flags": 4},
+            {"matrix": [4, 2], "x": 29, "y": 52, "flags": 4},
+            {"matrix": [4, 3], "x": 42, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 54, "y": 52, "flags": 4},
+            {"matrix": [4, 5], "x": 67, "y": 52, "flags": 4},
+            {"matrix": [4, 6], "x": 80, "y": 52, "flags": 4},
+            {"matrix": [4, 7], "x": 93, "y": 52, "flags": 4},
+            {"matrix": [4, 8], "x": 106, "y": 52, "flags": 4},
+            {"matrix": [4, 9], "x": 118, "y": 52, "flags": 4},
+            {"matrix": [4, 10], "x": 131, "y": 52, "flags": 4},
+            {"matrix": [4, 11], "x": 144, "y": 52, "flags": 4},
+            {"matrix": [4, 12], "x": 168, "y": 52, "flags": 1},
+            {"matrix": [4, 15], "x": 208, "y": 52, "flags": 1},
+            {"matrix": [5, 16], "x": 221, "y": 64, "flags": 1},
+            {"matrix": [5, 15], "x": 208, "y": 64, "flags": 1},
+            {"matrix": [5, 14], "x": 195, "y": 64, "flags": 1},
+            {"matrix": [5, 13], "x": 176, "y": 64, "flags": 1},
+            {"matrix": [5, 12], "x": 160, "y": 64, "flags": 1},
+            {"matrix": [5, 11], "x": 144, "y": 64, "flags": 1},
+            {"matrix": [5, 6], "x": 90, "y": 64, "flags": 4},
+            {"matrix": [5, 2], "x": 35, "y": 64, "flags": 1},
+            {"matrix": [5, 1], "x": 19, "y": 64, "flags": 1},
+            {"matrix": [5, 0], "x": 3, "y": 64, "flags": 1}
+        ],
+        "max_brightness": 120,
+        "sleep": true
+    },
+    "url": "www.yiancar-designs.com",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x4E55",
+        "vid": "0x8968"
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B0"
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_tkl_iso_tsangan"
+    },
```

1. layout alias not required
2. community layout support added
```suggestion
    "community_layouts": ["tkl_iso_tsangan"],
```